modules/apps/portal-language/portal-language-override-api/src/main/java/com/liferay/portal/language/override/exception/InvalidPropertiesFileException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.language.override.exception;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+/**
+ * @author Drew Brokke
```

Comment:
Hey, that's me! 😉

Reply:
Did it auto-generate with my name?

Reply:
haha yeah, your name is on the service.xml file